docs/changehistory/NextVersion.md

```diff
@@ -47,9 +47,86 @@ For the stop sign example described above, you might have a [glTF model](https:/
+
 Context reality models that have been attached using `DisplayStyleState.attachRealityModel`, can now be hidden by turning ON the `ContextRealityModel.invisible` flag.  Previous implementation requiered context reality models to be detached in order to hide it from the scene.
 
+### Contour Display
+
+A new rendering technique has been added to iTwin.js which allows a user to apply specific contour line renderings to subcategories within a scene.
+
+iTwin.js now provides the following API to use this feature:
+
+- [DisplayStyle3dSettings]($common) now has a `contours` property which contains all of the subcategories-to-styling association data necessary to enable this feature. That object is of type [ContourDisplay.Settings]($common).
+- [ContourDisplay.Settings]($common) defines how contours are displayed in the iModel based on a list of [ContourDisplay.Terrain]($common) objects in the `terrains` property.
+- [ContourDisplay.Terrain]($common) describes an assocation of subcategories to contour styling. It contains an array of subcategory IDs titled `subCategories`. Those subcategories will have the contour styling within the same terrain's [ContourDisplay.Contour]($common) `contourDef` object applied to them.
+- [ContourDisplay.Contour]($common) describes the rendering styling settings that apply to a specific set of subcategories within a [ContourDisplay.Terrain]($common). This actually describes stylings for two sets of contours: major and minor. These stylings are separate from each other. The minor contour occurs at a defined interval in meters. These intervals draw at a fixed height; they are not dependent on the range of the geometry to which they are applied. The major contour is dependent on the minor contour. The interval of its occurence is not measured directly in meters; rather it is a count of minor contour intervals between its occurrences. The properties describing how major and minor contours are styled are listed here:
+  - `majorColor` is the color that a major contour line will use. Defaults to [ColorDef.black]($common).
+  - `minorColor` is the color that a minor contour line will use. Defaults to [ColorDef.black]($common).
+  - `majorPixelWidth` is the width in pixels of a major contour line. (Range 1.5 to 9 in 0.5 increments). Defaults to 2.
+  - `minorPixelWidth` is the width in pixels of a minor contour line. (Range 1.5 to 9 in 0.5 increments). Defaults to 1.
+  - `majorPattern` is the pattern for a major contour line. Defaults to [LinePixels.Solid]($common).
+  - `minorPattern` is the pattern for a minor contour line. Defaults to [LinePixels.Solid]($common).
+  - `minorInterval` is the interval for the minor contour in the associated terrain in meters. Defaults to 1.
+  - `majorIntervalCount` is the count of minor contour intervals that define a major interval (integer > 0). Defaults to 5.
+
+Consult the following code for an example of enabling and configuring contour display in iTwin.js:
+
+```ts
```

Comment:
Before merging PR please move example code to the example-code-snippets package and reference it from here, to ensure it compiles now and in the future.

Reply:
Can you please provide an existing example of the iTwin-preferred style for how to reference a snippet from a NextVersion.md / numbered release markdown file?

Reply:
Snippet added. Linked within NextVersion.md. Reworked NextVersion.md to fit new API.